NEWS.md

```diff
@@ -177,7 +181,7 @@
 Google uses lintr heavily internally, and has developed a large set of linters improving code consistency
```

Comment:
Google heavily uses lintr internally ... ?

Reply:
I think the old order is more natural, but agree it's a bit clunky. tried paraphrasing instead.